examples/README.md

```diff
@@ -0,0 +1,84 @@
+# Centralized Telescope with BLS Signatures
+
+## Workflow: ALBA Threshold Signature Example
+1. **Define Telescope Parameters**:
+  - Set the number of elements (`nb_elements`) to 1,000.
+  - Specify `soundness_param` and `completeness_param` as 128.0 each.
+  - Calculate the prover set size (`set_size`) as 80% of `nb_elements` and the lower bound as 20%.
+  - Initialize the Telescope with these parameters.
+2. **Initialize Signers**:
+  - Create a list of candidate signers (`Signer`s) with randomly generated keys.
+3. **Start Registration**:
```

Comment:
@curiecrypt :  It may be even a good idea to overcommunicate with a simple example the registration - e.g., a total of 3000 SPOs but only, say,1000 of them register and the Mithril lottery is done out of these 1000. WDYT?